packages/universal-hooks/README.md

```diff
@@ -0,0 +1,35 @@
+# Universal Hooks
+
+This library aims to provide a common interface for React-like hooks to plug into. Simply plug in an implementation and your hooks are reusable across different implementations.
+
+## Usage
+
+```javascript
+// Set an implementation for the hooks
+setImplementation(haunted)
+// OR
+setImplementation(React)
+// OR
+setImplementation(Preact)
+
+// Use the hooks like normal
+function useCounter() {
+    const [counter, setCounter] = useState(0);
+
+    const increment = () => setCounter((c) => c + 1)
+    const decrement = () => setCounter((c) => c - 1)
+
+    return {counter, increment, decrement}
+}
+```
+
```

Comment:
Should have a section about Rollup/Webpack usage. People keep using this approach in our new components and get it wrong or it take several tries: https://gist.github.com/loganvolkers/591dade9a38b2fcf26f9c19d001ca325

Reply:
Done @ 6e2eebd